docs/BATCHING.md

```diff
@@ -66,11 +56,10 @@ In the undesirable cases where Solr becomes unresponsive (e.g., data corruption,
 
 If a Solr core has become unresponsive and the specified error threshold has been traversed, and if, in addition, the high water mark has been exceeded, then the yokozuna batching system has a mechanism for automatically purging enqueued entries, so as to allow vnodes to continue servicing requests, as well as to allow update operations to occur for indices that are not in a pathological state.
 
-The yokozuna batching subsystem supports 4 different purge strategies:
+The yokozuna batching subsystem supports 3 different purge strategies:
 
 * `purge_one` (default behavior): Purge the oldest entry from a randomly selected indexq structure among the set of search indexes which have crossed the error threshold;
 * `purge_index`: Purge all entries from a randomly selected indexq structure among the set of search indexes which have crossed the error threshold;
```

Comment:
Same as above - will purge all of the entries for the {index, partition} pair. ? is, should it actually purge the _whole index_ or is the current behavior of only purging the one solrq_worker "correct enough"? I believe it is, as this really previously would purge all the data for a single indexq structure _for a single worker_ as it does today.